6_drivers/in/AOSS_solar_README.md

```diff
@@ -0,0 +1,389 @@
+
+Quality assurance methods are in place to ensure against premature 
+equipment failure in the field and post deployment data problems.  For 
+example, all instruments at each station are exchanged for newly calibrated 
+instruments on an annual basis.  Calibrations are performed by world-
+recognized organizations.  For example, pyranometers and pyrheliometers are 
+calibrated at the National Renewable Energy Laboratory (NREL).  
+Calibration factors for the UVB instrument are transferred from three standards 
+maintained by SRRB's National UV Calibration Facility in Boulder.  In general 
+all of the standards the SRRB and NREL are traceable to NIST or its equivalent. 
+
+
+THE INSTRUMENTS:
+
+1.	The Solar Light UVB Broadband Radiometer
+
+The conversion factor applied to the UVB data is valid for the solar spectral band 
+from 280 to 320 nm and has a response function that approximates the erythemal (Diffey) 
+action spectrum.  The conversion factor applied varies as a function of the solar 
+zenith angle.  
+
+2.	The Normal Incidence Pryheliometer (NIP)
+
+The NIP measures direct solar radiation in the broadband spectral range from 
+280 to 3000 nm.  Those used at ISIS stations are calibrated nominally 
+on a biennial basis at the NOAA calibration facility in Boulder, CO
+
+3.	Precision Spectral Pyranometer (PSP)
+
+A PSP measures downwelling global irradiance at ISIS stations. 
+These instruments are sensitive to the same broadband 
+spectral range as the NIP, 280 to 3000 nm.  They are calibrated 
+on a biennial basis at the NOAA calibration facility in Boulder, CO
+
+An inherent problem with the PSP is that its sensor cools to space (if it
+is directed upward) and this causes a negative signal.  This is apparent at 
+night where it shows up as an erroneous negative irradiance  that can be as
+high as -30 Wm^-2.  Daytime data also contain this error but it is masked by 
+the large solar signal.  This error especially affects the diffuse measurement 
+made by a PSP because the sensor is shaded.  Therefore, the PSP is not used
+for the diffuse measurement.  
+
+4.	Eppley 8-48 "black and white" pyranometer 
+
+This pyranometer is used for the diffuse solar measurement.  It has been found 
+to have desirable properties for this measurement because it does not use a 
+thermopile for the detector and thus does not have the "nighttime" offset problem.  
+These instruments are sensitive to the same broadband spectral range as the NIP 
+and PSP, 280 to 3000 nm.  They are also calibrated on a biennial basis at 
+the NOAA calibration facility in Boulder, CO
+
+5.      Precision Infrared Radiometer (PIR) 
+
+The PIR measures and downwelling thermal infrared irradiance at Madison only.
+It is sensitive to the spectral range from 3000 to 50,000 nm.  NOAA maintains
+three standard PIRs that are calibrated annually by a world-reputable
+organization.  These standards are used to calibrate field PIRs.
```

Comment:
The file 6_drivers/in/ntl129_2_v5.csv is too big to view in GitHub. Is the repo the best place to be storing this file, or should it be moved to Drive?

Reply:
Yes, it should either be on drive or we should make a call to their server for the file w/ a fetch.